.gitignore

```diff
@@ -24,9 +24,11 @@ vendor/
 shared/modules/mocks/
 
 # Proto generated files
-types/
+*.pb.go
```

Comment:
Is this to force a regeneration? Is this necessary?

Reply:
Yup.